*** DEVELOPMENT ENVIRONMENT ***
Task: Save results spreadsheet
Error: Something went wrong:
Error reading model: E0 (Error): XML content (core, L22); Badly formed XML; XML content is not well-formed.

Traceback (most recent call last):
  File "/home/alex/.local/lib/python3.7/site-packages/cobra/io/sbml.py", line 210, in read_sbml_model
    **kwargs)
  File "/home/alex/.local/lib/python3.7/site-packages/cobra/io/sbml.py", line 288, in _sbml_to_model
    raise CobraSBMLError("No SBML model detected in file.")
cobra.io.sbml.CobraSBMLError: No SBML model detected in file.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "../../findCPcore_pkg/findCPcore/CobraMetabolicModel.py", line 325, in read_model
    self.__cobra_model = cobra.io.read_sbml_model(path)
  File "/home/alex/.local/lib/python3.7/site-packages/cobra/io/sbml.py", line 217, in read_sbml_model
    "Something went wrong reading the SBML model. Most likely the SBML"
cobra.io.sbml.CobraSBMLError: Something went wrong reading the SBML model. Most likely the SBML model is not valid. Please check that your model is valid using the `cobra.io.sbml.validate_sbml_model` function or via the online validator at http://sbml.org/validator .
	`(model, errors) = validate_sbml_model(filename)`
If the model is valid and cannot be read please open an issue at https://github.com/opencobra/cobrapy/issues .

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/alex/findCPOrganization/findCPCLI_pkg/scripts/findCPcli", line 387, in <module>
    main()
  File "/home/alex/findCPOrganization/findCPCLI_pkg/scripts/findCPcli", line 363, in main
    run(task, input_file, out_path, verbose, objective, fraction)
  File "/home/alex/findCPOrganization/findCPCLI_pkg/scripts/findCPcli", line 175, in run
    raise error
  File "/home/alex/findCPOrganization/findCPCLI_pkg/scripts/findCPcli", line 109, in run
    objective=objective,
  File "../../findCPcore_pkg/findCPcore/Facade.py", line 128, in generate_spreadsheet
    self.spreadsheet = f.run_summary_model(model_path, print_f, args1, args2, objective, fraction)
  File "../../findCPcore_pkg/findCPcore/FacadeUtils.py", line 267, in run_summary_model
    model = CobraMetabolicModel(model_path)
  File "../../findCPcore_pkg/findCPcore/CobraMetabolicModel.py", line 302, in __init__
    self.read_model(path)
  File "../../findCPcore_pkg/findCPcore/CobraMetabolicModel.py", line 360, in read_model
    raise Exception("Error reading model: " + error_list[0])
Exception: Error reading model: E0 (Error): XML content (core, L22); Badly formed XML; XML content is not well-formed.

